test/test-utils/console.ts

```diff
@@ -26,11 +26,11 @@ const originalFunctions: FilteredConsole = {
 
 /**
  * Allows to filter out specific messages in console.*.
+ * Automagically restores the original function by implementing an afterAll hook.
  *
  * @param ignoreList Messages to be filtered
- * @returns function to restore the console
  */
-export const filterConsole = (...ignoreList: string[]): (() => void) => {
+export const filterConsole = (...ignoreList: string[]): void => {
     for (const [key, originalFunction] of Object.entries(originalFunctions)) {
```

Comment:
Would it make sense to wrap this in `beforeEach()` or `beforeAll()`?

Reply:
Clean up is done in `afterAll` → going to put this into a `beforeAll` :+1: